lib/travis/api/v3/service.rb

```diff
@@ -89,21 +89,23 @@ def result_type
       self.class.result_type
     end
 
-    def result(*args)
-      Result.new(access_control, *args)
+    def result(resource, **meta_data)
+      return not_found unless resource
+      rt = meta_data[:result_type] || result_type
+      status = meta_data[:status] || 200
```

Comment:
If meta_data includes it yes. This was happening with the payload that was coming from the accepted method below. The payload in that method has status specified and it explicitly sets the status, which was a bit confusing as to the intent.